applications/ndk/1.3.0/release/ndk.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: OCIRepository
+metadata:
+  name: ndk
```

```suggestion
  name: ndk-1.3.0-source
```
adding version into the OCIRepository ensures upgrades are not racy. (With current naming, a newer version of ndk helmrelease can use an older version of ocirepository.

---

Great suggestion, thanks for pointing this out. :)